internal/k8s/ako_init.go

```diff
@@ -276,6 +278,10 @@ func (c *AviController) InitController(informers K8sinformers, registeredInforme
 	} else {
 		// First boot sync
 		err = c.FullSyncK8s()
+		ctrlAuthToken := os.Getenv("CTRL_AUTHTOKEN")
+		if ctrlAuthToken != "" {
+			c.RefreshAuthToken()
```

Comment:
What happens if something goes wrong while refreshing the auth token? Are we taking care of restarting AKO?

Reply:
if the refresh fails even with the retries, then we wait for the next cycle after 12 hours...

Reply:
What I meant here was, we need to handle the error from `RefreshAuthToken` and restart AKO, if there was an error.